peachjam/templates/peachjam/user_following_delete.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+<form hx-post="{% url 'user_following_delete' object.pk %}?{{ request.GET.urlencode }}">
```

Comment:
is the query string part necessary?

Reply:
yes, we use that to reload the button after unfollow.

Reply:
of course! 🙏